Find a functionally equivalent code:android.printservice.PrintJob.getAdvancedStringOption:COMMENT
Method Modifier: public      
Comment:/**
 * Gets the value of an advanced (printer specific) print option.
 *
 * @param key The option key.
 * @return The option value.
 */

@@ -1,4 +1,4 @@
 {
     PrintService.throwIfNotCalledOnMainThread();
-    return null;
+    return getInfo().getAdvancedStringOption(key);
 }

